apps/condo/domains/miniapp/schema/B2BAppPermission.test.js

```diff
@@ -189,8 +190,9 @@ describe('B2BAppPermission', () => {
                 })
             })
             test('No one in the organization can, including the administrator', async () => {
-                const allPermissions = await B2BAppPermission.getAll(manager, { id: permission.id })
-                expect(allPermissions).toHaveLength(0)
+                await expectToThrowAccessDeniedErrorToObjects(async () => {
+                    await B2BAppPermission.getAll(manager, { id: permission.id })
```

Comment:
why error on `read` operation?

Reply:
That's default keystone behaviour then access is resolved to `false`. Previously we returned a filter, which was making no sense (and that's why returns empty results). So, this is more general "keystone" behaviour, not part of this PR